server.go

```diff
@@ -82,6 +82,7 @@ type service struct {
 	server interface{} // the server for service methods
 	md     map[string]*MethodDesc
 	sd     map[string]*StreamDesc
+	meta   interface{}
```

Comment:
I am not a native speaker. But to me "meta" is a prefix in English. I do prefer a noun instead of a language prefix here. And I do not think "metadata" is long.